tools/make_patch.c

```diff
@@ -464,13 +463,17 @@ int get_constant(const char *filename, const char *constant)
 	return value;
 }
 
-void interpret_command(char *command, const struct symbol *symbol, struct patch *patch,
-                       FILE *new_rom, FILE *orig_rom, FILE *output)
+void interpret_command(char *command, const struct symbol *lastFoundSymbol, const struct symbol *symbols, struct patch *patch,
+						FILE *new_rom, FILE *orig_rom, FILE *output)
```

Comment:
Tabs for indentation, spaces for alignment... Keep it as it was.

Reply:
Replaced tabs with spaces and moved part of the line down in https://github.com/pret/pokecrystal/pull/882/commits/3a78bcc8073928f2675d66b38124b90b6796ffca